blob/azureblob/azureblob_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azureblob
+
+import (
+	"os"
+	"context"
+	"net/http"	
+	"testing"
+	
+	"github.com/google/go-cloud/blob/driver"
+	"github.com/google/go-cloud/blob/drivertest"
+	"github.com/google/go-cloud/internal/testing/setup"
+)
+
+// Prerequisites for --record mode
+// 1. Sign-in to your Azure Subscription and create a Storage Account
+//    link to the Azure Portal: https://portal.azure.com
+// 2. Locate the Access Key (Primary or Secondary) under your Storage Account > Settings > Access Keys
+// 3. Set Environment Variable AZURE_STORAGE_ACCOUNT_KEY below
```

Comment:
Instead of requiring code changes, can we do this from the environment? How does Azure do this normally? Both GCP and AWS have a default way to load credentials (off of some standard location on disk).

Reply:
Fixed in [changeset](https://github.com/google/go-cloud/pull/936/commits/8af9c6ea4a676fc9586770c64f07eb02e1339d80); added -settingsfile flag for credentials